.env.dev

```diff
@@ -5,3 +5,5 @@ VW_DB=sqlite://./db.sqlite
 VW_OPENID_CLIENT_ID=dev-client
 VW_OPENID_CLIENT_SECRET=public-secret
 VW_OPENID_SCOPE=openid profile
+VW_ORG_NAME=VINYVAJI
```

Comment:
Vinywaji

Reply:
Fixed the typo.
Yeah. For the Bit-Bots like design you have to set `VW_THEME_COLOR=amber` in your environment.